modules/steam.nix

```
@@ -439,5 +471,79 @@ in
         WINE_CPU_TOPOLOGY = "8:0,1,2,3,4,5,6,7";
       };
     })
+    (mkIf cfg.autoStart {
+      assertions = [
+        {
+          assertion = !config.systemd.services.display-manager.enable;
+          message = "No Display Managers must be enabled when jovian.steam.autoStart is used";
```

```suggestion
          message = ''
            Traditional Display Managers cannot be enabled when jovian.steam.autoStart is used
            
            Hint: check `services.xserver.displaymanager.*.enable` options in your configuration.
          '';
```

I wonder if we could even detect which one is enabled for the assertion message, but that could be filed as an issue for future improvement.